packages/dev/core/src/Cameras/Inputs/arcRotateCameraMouseWheelInput.ts

```diff
@@ -202,6 +210,17 @@ export class ArcRotateCameraMouseWheelInput implements ICameraInput<ArcRotateCam
         this._hitPlane = Plane.FromPositionAndNormal(camera.target, direction);
     }
 
+    private _updateCameraTarget() {
+        const camera = this.camera;

```

Comment:
All the functions used here should be toRef and Garbage Collection Free. currently substract, Plane, Ray and so on are all creating lot of GC pressure every frames.

Reply:
> All the functions used here should be toRef and Garbage Collection Free. currently substract, Plane, Ray and so on are all creating lot of GC pressure every frames.

Use subtractToRef、_ray、_hitPlane instead